Sync/BookmarkPayload.swift

```diff
@@ -352,8 +352,8 @@ open class BookmarkPayload: BookmarkBasePayload {
     }()
 
     lazy var tagsString: String = {
-        if self["tags"].type == Type.array {
-            return self["tags"].stringValue
+        if self["tags"].isArray() {
+            return self["tags"].stringValue() ?? "[]"
```

Comment:
`isArray` and `isDictionary` are safe (well, relatively safe) — the type code is an accurate determination of whether calling `.array` will give you an array of `JSON`.

The changes on these two lines are actually in two commits to make reviewing this easier!

The change on line 356 is from `.stringValue` (which gives you the string value of a `JSON` string if there is one, and thus always returns `""` for this array) to `.stringValue()` (which JSON-serializes a complete JSON structure, in this case an array).

We should rename our `stringValue` method to something that isn't so typo-close to a built-in var, but I didn't want to do this here. IIRC this was one @justindarc or @sleroux introduced as a wrapper around SwiftyJSON's serialization call.